src/ModelObjects/xml_parser.rs

```diff
@@ -186,7 +185,7 @@ fn parse_declarations(variables: &str) -> Declarations {
                 } else {
                     let mut error_string = "not implemented read for type: ".to_string();
                     error_string.push_str(&variable_type.to_string());
-                    panic!(error_string);
+                    panic!("{}", error_string);
```

Comment:
Im confused what is the purpose of this change, does it make some change to how its handled internally?

Reply:
The first argument to `panic!` is a format string meaning any occurrences of `{}` would be attempted to be replace by a value. By changing it to this we guarantee that all error strings will be printed as-is.